cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml

```diff
@@ -114,27 +114,27 @@ spec:
     rules:
     - alert: CephOSDCriticallyFull
       annotations:
-        description: Utilization of back-end storage device {{ $labels.ceph_daemon
-          }} has crossed 85% on host {{ $labels.hostname }}. Immediately free up some
-          space or expand the storage cluster or contact support.
+        description: Utilization of storage device {{ $labels.ceph_daemon }} of device_class
+          type {{$labels.device_class}} has crossed 80% on host {{ $labels.hostname
```

Comment:
looks slightly verbose for my taste. something like "Storage utilisation on {osd} of {device_type} has crossed 80%. Please free up space or add capacity of {device_type} "  would be slightly better in tone

Reply:
@vbnrh I have ket it verbose so that users understand- 

- Which device has crossed the threshold and what is an osd ? ( Not everyone knows what an osd is )
- On which node it is
- And what is the type of it.

I feel its better to keep things explicit than implicit.